src/main/java/org/saltyrtc/client/signaling/InitiatorSignaling.java

```diff
@@ -598,8 +598,7 @@ Peer getPeerWithId(short id) throws SignalingException {
         if (id == SALTYRTC_ADDR_SERVER) {
             return this.server;
         } else if (this.isResponderId(id)) {
-            //noinspection ConstantConditions
-            if (this.getState() == SignalingState.TASK && this.responder != null && this.responder.getId() == id) {
+            if ((this.getState() == SignalingState.TASK) && (this.responder != null) && (this.responder.getId() == id)) {
```

Comment:
Can you remove the parentheses? They add unnecessary noise.

Reply:
Ups, I like those parentheses because you don't have to think about the operation order.

Will change though. :smile: